src/metatensor/models/utils/data/dataset.py

```diff
@@ -18,8 +18,9 @@ class TargetInfo:
     """
 
     quantity: str
-    unit: str
+    unit: str = ""
     per_atom: bool = False
+    gradients: List[str] = field(default_factory=list)
```

Comment:
can you add some docs here on what this "gradients" means? From reading the code, it seems that this is to put the set of gradients contained in the dataset, which don't necessarily match which gradients should be used by the model for training, or which gradients should be predicted.